src/pages/patientView/PatientViewUrlWrapper.ts

```diff
@@ -47,4 +51,15 @@ export default class PatientViewUrlWrapper extends URLWrapper<
     public setTab(tab: string): void {
         this.updateURL({}, `${PagePath.Patient}/${tab}`);
     }
+
+    @computed public get tabId() {
+        const regex = new RegExp(`${PagePath.Patient}\/(.+)`);
```

Comment:
seems like a better strategy to split on slash and then use index.  That would hew better to the definition of the path protocol and also handle a future case where we might introduce deeper paths.